packages/cli/src/modulars/moduleLoader.ts

```diff
@@ -39,11 +42,23 @@ export function loadDependency<N extends NETWORK_FAMILY>(network: N): ModuleCach
   }
   //Check dependencies actual satisfy INetworkCommonModule
   if (
-    moduleCache[network].parseProjectManifest === undefined ||
-    moduleCache[network].isCustomDs === undefined ||
-    moduleCache[network].isRuntimeDs === undefined
+    moduleCache[network]?.parseProjectManifest === undefined ||
+    moduleCache[network]?.isCustomDs === undefined ||
+    moduleCache[network]?.isRuntimeDs === undefined
   ) {
     throw new Error(`${packageName} is not compatible, please make sure package update to latest version`);
   }
   return moduleCache[network];
 }
+
+export function loadEthModule(): EthereumNetworkModule {
+  const ethModule = loadDependency(NETWORK_FAMILY.ethereum);
+  assert(ethModule, 'Eth module got undefined, expected to be loaded from local or global');
```

Comment:
This assertion should be redundant, `loadDependency` should never return undefined